engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -1408,13 +1406,12 @@ struct void_torrent_t final : public priest_spell_t
     energize_type     = action_energize::PER_TICK;
     energize_resource = RESOURCE_INSANITY;
     energize_amount   = insanity_gain;
-
-    child_dp->background = true;
   }
 
+  // DoT duration is fixed at 3s
   timespan_t composite_dot_duration( const action_state_t* ) const override
```

Comment:
Could this be achieved without the override, by maybe setting some hasted=false flags?

Reply:
The spell is still hasted tickwise, just fixed duration channel.

Reply:
@scamille we need hasted ticks but fixed duration, dont think thats an option that i can see